frontend/src/components/common/table/Pagination.tsx

```diff
@@ -44,7 +44,7 @@ const Pagination = ({
             &lt; Previous
           </PaginationPrevious>
         )}
-        <PaginationPageGroup align="center" isInline>
+        <HStack align="center">
```

Comment:
In order to maintain semantic HTML I think we should use the library's built-in components (according to [here](https://github.com/niconiahi/ajna/blob/main/packages/pagination/src/components/PaginationPageGroup.tsx) the component already sets `isInline` so we sidestep the deprecation), or else duplicate all the other props that the library already sets.